src/routers/createConfigGetter.js

```diff
@@ -86,5 +87,7 @@ export default (
     outputConfig = applyConfig(componentScreenConfig, outputConfig, configOptions);
     outputConfig = applyConfig(routeScreenConfig, outputConfig, configOptions);
 
+    validateScreenOptions(outputConfig, route);
```

Comment:
@ericvicenti we should probably `if` on node_env here so that this can get removed when bundling?